src/coprocessor/codec/error.rs

```diff
@@ -66,6 +67,17 @@ quick_error! {
 }
 
 impl Error {
+    pub fn handle_invalid_time_error(ctx: &mut EvalContext, err: Error) -> Option<Error> {
```

Comment:
How about just return a `Result<()>`, so that you can just use `?` operator when using it?

Reply:
fix